.github/workflows/deploy-to-Netlify.yml

```diff
@@ -0,0 +1,12 @@
+name: Deploy site to Netlify
```

Comment:
pls rename to `deploy.sh`
we don’t have to specify deploy platform as we have only one option

Reply:
```suggestion
name: Deploy application
```

Reply:
Done